configsamples/bgppeer_withsecret.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
   holdTime: 9m0s
   keepaliveTime: 0s
+  connectTime: 120s
```

Comment:
this seems to be a bit hidden, let's add a new file "_with_session_options"

Reply:
done :)